### ISSUE TITLE
Figure out a better way to get pubDate

### ISSUE THREAD

**Issue by williamspiro** [williamspiro/hubXml#16]
Right now, I am hitting `http://www.convert-unix-time.com/api` which is less than ideal

**Comment by williamspiro** [comment]
http://natty.joestelmach.com/

**Comment by Jazilch** [comment]
@williamspiro I took a look into this too. I believe you're correct that your best bet is to use Natty library when you linked above. This [stack overflow article](https://stackoverflow.com/questions/3389348/parse-any-date-in-java) seems to support your thought process here.   I wasn't seeing a good way either to convert or format dates without first knowing exactly the date format. 

This is a simple example I found of using the Natty library if that helps too but looks like that is your best option. 
```
public static Date parse(String date) throws Exception {

    try{

        List<DateGroup> parse = new PrettyTimeParser().parseSyntax(date);
        return parse.get(0).getDates().get(0);

    }catch (Exception e){
        throw new Exception("unparseable date: " + date);
    }

}
